bear-requirements.yaml

```diff
@@ -1,3 +1,84 @@
+# This is a comment!
+overrides: coala-build.yaml
+pip_requirements:
```

Comment:
can you modify the generator so this doesnt move in the yaml.
It is easier to verify the code is correct than verify the moved yaml is identical

Reply:
What do you mean? The pip_requirements part? or the overrides part?